corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -5,98 +5,98 @@
-                <i class="fa fa-cloud-upload"></i>
-                {% trans "Import from Excel" %}
-            </a>
-            <button data-bind="click: $root.showDeprecated, visible: !showAll()" class="btn btn-default pull-right">{% trans "Show Deprecated Properties" %}</button>
-            <button data-bind="click: $root.hideDeprecated, visible: showAll" class="btn btn-default pull-right">{% trans "Hide Deprecated Properties" %}</button>
-            <table id="data-dictionary-table" class="table">
-                <thead>
-                    <th></th>
-                    <th>{% trans "Case Property" %}</th>
-                    <th>{% trans "Data Type" %}</th>
-                    <th>{% trans "Description" %}</th>
-                    <th></th>
-                </thead>
-                <tbody data-bind="sortable: casePropertyList">
-                    <!-- ko if: isGroup -->
-                    <tr class="active">
-                        <td><i class="fa" data-bind="css: { 'fa-plus-square': !expanded(), 'fa-minus-square': expanded() }, click: $root.toggleGroup"></i></td>
-                        <td data-bind="text: name ? name : 'No Group'"></td>
-                        <td>{% trans "Case Property Group" %}</td>
-                        <td></td>
-                        <td></td>
-                    </tr>
-                    <!-- /ko -->
-                    <!-- ko if: !isGroup -->
-                    <tr data-bind="visible: expanded() && (!deprecated() || $root.showAll()), attr: {'data-order': _sortableOrder}">
-                        <td class="sortable-handle">
-                            <i class="fa fa-arrows-v"></i>
-                        </td>
-                        <td data-bind="text: name"></td>
-                        <td>
-                            <select class ="form-control"
-                                    data-bind="
+  {% registerurl 'data_dictionary_json' domain %}
+  {% registerurl 'update_case_property' domain %}
+  {% initial_page_data 'typeChoices' question_types %}
+  <div data-bind="saveButton: saveButton"></div>
+  <div class="row">
+    <div class="col-md-10">
+      <h3 data-bind="text: $root.activeCaseType()"></h3>
+      <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">
+        <i class="fa fa-cloud-download"></i>
+        {% trans "Export to Excel" %}
+      </a>
+      <a class="btn btn-default" href="{% url "upload_data_dict" domain %}">
+        <i class="fa fa-cloud-upload"></i>
+        {% trans "Import from Excel" %}
+      </a>
+      <button data-bind="click: $root.showDeprecated, visible: !showAll()" class="btn btn-default pull-right">{% trans "Show Deprecated Properties" %}</button>
+      <button data-bind="click: $root.hideDeprecated, visible: showAll" class="btn btn-default pull-right">{% trans "Hide Deprecated Properties" %}</button>
+      <table id="data-dictionary-table" class="table">
+        <thead>
+        <th></th>
```

Comment:
Possible that pycharm doesn't recognize <th> as needing to be indented? Seems weird as it looks like pycharm is right about most other things

Reply:
yeah...in later PRs i've been adjusting these things manually...also `<span>` elements.